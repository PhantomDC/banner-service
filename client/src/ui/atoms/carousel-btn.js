import React from 'react'
import styled from 'styled-components'


export const CarouselBtn = styled.div`
  display: block
`
