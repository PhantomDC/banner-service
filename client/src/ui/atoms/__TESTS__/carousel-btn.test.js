import React from 'react'
import { configure, render, shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import { create } from 'react-test-renderer'
import { CarouselBtn } from '../carousel-btn'

/* eslint-disable no-undef */
/* eslint-disable function-paren-newline */

configure({ adapter: new Adapter() })

describe('test component CarouselBtn', () => {
  it('CarouselBtn render without crash', () => {
    render(<CarouselBtn />)
  })

  it('isLeft props test', () => {
    const wrapper = shallow(<CarouselBtn isLeft />)
    const { isLeft } = wrapper.instance().props

    expect(isLeft).toBeTruthy()
  })

  it.skip('shapshot Button', () => {
    const tree = create(<CarouselBtn isLeft />)
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})
