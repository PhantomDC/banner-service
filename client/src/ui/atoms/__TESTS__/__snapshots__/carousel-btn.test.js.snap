// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test component CarouselBtn shapshot Button 1`] = `
<div
  className="sc-bdVaJa cpZBuY"
/>
`;
